references/dorks/cc-parameters-lowercase-all.txt
id
page
p
action
share
utm_source
page_id
utm_medium
view
utm_campaign
lang
redirect_to
add-to-cart
s
pid
cat
replytocom
lightbox
type
itemid
m
mode
start
fbclid
q
option
t
d
msg
sort
shared
tmpl
tag
fp
prvtof
c
url
u
poru
print
link
post_type
elementor_library
paged
catid
a
attachment_id
category
f
route
subid1
utm_content
order
author
_wpnonce
cid
ref
product_id
returnurl
layout
year
template
title
source
redirect
task
add_to_wishlist
month
showcomment
from
tid
do
r
amp
format
gclid
uid
like_comment
language
path
sid
key
controller
feed
utm_term
v
ical
et_blog
keyword
search
file
o
name
mod
variant
code
term
category_id
date
orderby
no
chash
tab
filter
mid
destination
l
pg
act
aspxerrorpath
gid
i
aid
limit
fid
product
dn
domain
n
bo_table
module
taxonomy
filter_by
cate_no
preview
token
limitstart
return
board_no
products_id
next
dir
g
width
search_id
section
topic
show
wr_id
offset
id_product
locale
tribe_event_display
query
day
folio
unapproved
height
moderation-hash
_ga
y
location
product_no
reauth
information_id
tribe_paged
back
op
goto
display_group
target
kw
item
cpath
src
product_count
e
site
k
product_order
model
rch
nid
add
login
user
idx
pageid
time
sort_by
b
menu
instance_id
country
state
rfolio
_glst
oldid
brand
keywords
ts
wd
area
x
w
product_cat
returnto
to
sortby
st
article_id
tags
post_id
main_page
step
status
city
yr
classid
new
categoryid
make
username
price
tt
phpsessid
content
objectpath
host
doing_wp_cron
method
redlink
sr
userid
hl
csid
bid
display
ga
cbid
pagen_1
data
album
h
product_orderby
gerf
guro
manufacturer_id
put
size
nonce
_ss
_pos
cat_id
ds
lng
dm
num
color
author_id
currency
direction
download
ty
printable
viewaction
board
sa
product_view
da
ma
referer
cmd
remove_item
max-results
thread
parms
hstr
cbrehoppp2
redirect_uri
newsid
hash
event
error
pagina
kt
service
image
event_id
subject
channel
client_id
nd
product-page
tabid
sub
return_url
kp
post
lid
ki
news
iframe
kld
ktd
maxads
password-protected
user_id
kbetu
app
sort_order
returntoquery
id_category
nomemberorder
page_number_0
r2
no_cache
response_type
course
comefrom
typeid
ac
shop_id
pag
vid
highlight
updated-max
stx
article
theme
nav
list
region
did
per_page
news_id
sd
sop
group
mailpoet_page
sfl
portalid
go
tipo
cate
reset
sk
mo
eventdisplay
comment
referrer
diff
robots
form
style
searchword
level
max
oscsid
mc_cid
item_id
sst
reason
sc
mc_eid
sod
eid

oid
extra
portfolio
ct
submit
sca
cp
rid
scope
provider
searchid
rp
mobile
p_p_id
typ
p_p_lifecycle
skip
p_p_state
letter
id_cms
aspxautodetectcookiesupport
_sid
p_p_mode
pp
number
plugin
1
index
products-per-page
productid
pos
info
portfoliocats
part
img
text
func
gallery
ajaxcalendar
version
register
registration
wysijap
zoom
kbc
yprpnd
filename
categoria
archive
siteid
return_to
desc
current
disabled
showall
_
yep
gtnp
gtpp
aff
md5
albumid
dispatch
manufacturer
qty
is_comment
context
_page
device
sort_method
hidetrans
ver
sordernumber
readmore
class
sinquiry
all
ns
document_srl
create_account
get
redirect_url
back_url
preview_id
pop
guid
pr
blackhole
folder
hidelinks
map
groupid
itsec-hb-token
rev
eventid
message
hslang
parent
lg
id_article
view_type
z
pageno
max_price
markreadhash
preview_nonce
subcat
pagenum
ws
na
search_type
min_price
gvbodystyle
class2
fn
urltype
element_id
pn
org
loc
relaystate
samlrequest
attredirects
kat
value
spread_flag
myaccount_hash
art
min
urec_id
media
selectedaccountid
_hsenc
prec_id
rss
hideredirs
forum
no_html
lat
rtype
co_id
tierand
items_per_page
cftoken
pagename
sn
tb_iframe
order_by
articleid
cs
by-date
dl
cl
deptid
webshielddrsessionverify
sp
prev_next
cfid
cpage
rating
seite
campaign
detail
component
gclsrc
backurl
price_max
md
refpageviewid
ajax
coid
embed
ss
slug
faq
loja
price_min
email
dy
pt
sec
board_act
beds
hd
form_id
seq
dd
searchtype
storyid
quantity
id_lang
redirecturl
olspage
hid
identifier
platform
idioma
productcateid
advanced
_rpo
_rdm
filter_brand
product_category
responsesopen
sectok
sitemap
group_id
count
team
sku
section_id
page2
je
owner
pic
srp
sra
menuid
origin
__hstc
prod
__hssc
auth
utm_referrer
wbtreeid
video
skuid
shop_view
member
cc
la
rb.routing.mode
rb.routing.signature
srk
edit
perpage
sourceid
categories
page_4
tp
kind
__hsfp
404
blog_id
fc
fontsize
doc
j
filters
days
popup
iccaldate
ctl
_thumbnail_id
srsrc
series
field
contenttype
random-post
flypage
baths
postid
rec_id
pdf
menu_id
refer
per_row
mcode
replyto
order_type
subid
collection
pageindex
find
flag
showtopic
mact
period
2
zipmin
special
inline
redir
pages_id
autoplay
calendar_page
mini
seesaa_related
_a
rewrite
mcat
function
issue
product_sort
pagesize
pagenumber
rubrik
ids
brd
fuseaction
word
lan
manufacturers_id
href
filter_color
mobileaction
pagetype
page_num
project
wid
content_id
ad
album_id
page_no
c_id
digest
class1
localenddate
localstartdate
introid
plan
commentid
fontfamily
fontcolor
sectionid
backgroundcolor
wap2
rt
in
p_id
county
spref
os_destination
page_6
job
genre
viewmode
full
nats
bidid
page3
productcode
ri
customize_changeset_uuid
avia-element-paging
container
cod
sortdirection
spm
xcode
iid
postorder
uri
postdays
o1
ie
sayfa
tcb_lightbox
hop
search_query
sl
ln
noamp
so
vehicleid
ps
per-page
postcount
portal_id
contact
try
rel
condition
zone
str
mobileon
type_0
idp
week
tb
partner
constraint
nr
product_tag
open
set_filter
entry
evid
banner
store
types
cd
clearall
showposts
objectid
pno
dt
profile
sortorder
3
admin
db
sortid
season
table
position
mc_id
gender
wmc-currency
photo
promotionid
selected
link_product_no
message_id
home
type_id
slideshow
subaction
caltype
mark
eo_month
command
item_name
at
all_items
uuid
et_fb
attachauth
contentid
_m
dealer
_hsmi
site_id
bmode
session
zip
top
kategorie
wishlist-action
viewobjectpath
forgot_password
serverid
scode
id_rubrique
on
quick
came_from
sf_paged
cntnt01returnid
tab_details
blog
upage
ptype
ordering
ip
tagname
ca
rpp
ft
item_description
target_object_id
item_pubdate
photoid
branduid
message_icon
0
ca_id
qid
4
tpl
address
output
pc
max_items
reload_once
skin
wpv_view_count
cb
lastviewaction
member_id
cur
mls
alias
line
param
tribe-bar-date
phocaslideshow
arc
combine
modus
keys
categorie
idcat
vin
calendar_page_prev
ym
kid
what
shop
source_id
label
tab_files
showagency
promo
company
download_id
ipa
sort_index
sb
pgid
node
rd
age
fileticket
_route_
opendocument
desktop
starteronly
nocache
client
csrfkey
uin
mt
it_id
link_id
stype
description
class_id
article_category
eventviewmode
loginsocial
reviews_id
showall_1
ord
noquote
wa
veaction
idxno
swoof
modal
ch
ai1ec
place
pa
wpformidentifier
suri
pwd
setcurrencyid
_pp
channel_id
pagespeed
radius
hidenotice
pi
eventlistviewmode
range
hsctatracking
5
marker
ml
dest
my_post_key
main
product_slno
event2
amount
tribe_venue
btag
p_l_id
qs
event1
id_product_attribute
signature
newskeyid
eunix
sunix
calendar_feature_id
refresh
searchphrase
browse
expand
entryid
noredirect
cms_block
results
rv_pid
cdp
docid
alt
ext
af
grid_list
usg
lc
culture
sm
kwd
pf
wpv_paged
marca
ef_id
event3
pagen_2
redirected
end
variation_id
mediatype
p1
co
bigclassname
origem
loca
quote
sf
lookup
calendar
adid
photo_id
reg
customize_autosaved
lp
tri
club_id
locn
player
wtrealm
second_id
pnum
coupon
hot
r_done
full-site
by
appid
response_mode
play
origin_path
islogin
feature
ci
set
contrast
division
templateid
wpbdp_view
s1
wctx
formid
tc
g2_itemid
topic_id
body
searchthreadid
category_name
affid
a_aid
gfdt
scid
feedformat
sorting
source_tax
_id
daysprune
ertthndxbcvs
wuid
_bc_fsnf
channelid
prodid
info_id
clear
_action
viewid
submissionid
vs
6
ms
prefix
hidebots
pl
id_manufacturer
room
tree
_nodeselectname
_nojs
first
fr
ano
pbminit
filter_size
tm
offer
book
an
p_p_auth
listing_id
random
navid
lanmu
moduleinstanceid
request
logout
elqtrackid
codigo
custom_date_start
rfsn
long_events
domainid
custom_date_end
change_section
where
hledej
shortcode
department
select
searchsubmit
view_mode
lm
flexdataid
artist
renderloc
ccode
continue
asc
ni
idtype
calid
var
comments
screen
ctype
etal
wlfrom
attachmentid
threadid
categ
7
subscribe_to_grandchildren
langue
strona
track
elementskit_template
archiv
hideminor
fs
game
pk_campaign
virtuemart_category_id
rc
_jcp
class3
utf8
ae_global_templates
rct
school
set_language
execution
cnid
id2
seccion
tag_id
lei
stock
ls
rate
sfvrsn
select_arrange
tracelog
sesskey
hideliu
advancedsearchform
com_board_id
formhash
p2
check
jobid
material
boardid
ai1ec_post_ids
isbn
phocadownload
savefacetstate
subcategory
catalogid
hidemyself
ka
total
permissionviolation
mm
sorttype
original
idc
8
sigalg
distance
sn1
searchparam
base_id
se
autor
strana
divpage
err
elqaid
image_id
idprod
sortfield
rub
elqat
pmid
module_id
fl
mepr-unauth-page
slide
hideanons
tagid
thread_id
com_board_basic
artid
storeid
cart
zmena-vzhledu
rq
vmcchk
location_id
starts_with
cts
lon
style_id
debut_articles
customize_messenger_channel
offer_token
showforum
pname
pg_avancada
board_code
no_results
oceanwp_library
req
mon
promocode
modid
featured
center
lost-password
actionid
blogid
wpdmdl
columnid
filtertype
neighborhood
jet-theme-core
tamanho
sale
operation
affiliate
cinfo
9
filter_reset
rf
cor
document
com_board_idx
mc
mkt_tok
cntnt01articleid
hauth.start
mapa-webu
hauth.time
incident_id
cate_id
post_format
instance
hilit
col
g2_view
edet
xinfo
rpinfo
page_number_1
vp_filter
cgid
details
_49_groupid
_49_privatelayout
id_cms_category
anchor
object_id
wpdestfile
liststyle
personid
checkout
ind
swudnsai
alpha
talle
trackback
max_height
max_width
rno
agent
languageid
dmn
incentiveid
ska
cookie
number_of_uploads
i_columnid
sort_key
i_webid
langid
refid
opt
orderway
lastfacetinteracted
pages
page1
pcode
ic
ctrl
account
handlekey
showuser
chapter
maxwidth
ed
si
sphrase_id
directorytype
pagination
volume
tmad
header
tmcampid
c1
maxheight
jid
mn
prev_url
transactionid
parentpagealias
occurrence
redirectto
sc_section_code
topicseen
xid
srchtxt
more
amid
result
utm
rnd
ved
forum_id
mailform
rotate
com_board_page
rubrique1
myteam
font
tip
resultsperpage
cont
parent_id
productsublayout
_gac
bigclassid
rurl
s2
pub
case
sel
datecut
ignoreredirect
memberid
catalog
vehicle_id
pk
object
pda
minprice
params
base
reverse-paginate
viewfull
folder_id
wcm_redirect_id
wcm_redirect_to
pagenotfound
portfolio_category
bedrooms
from_id
content_only
sortoption
pgnr
category_no
tmclickref
topicid
aff_id
idn
start_date
_artperpage
file_id
rs
et
products
upn
sort_type
gadget
lastname
listing_type
savelastpath
wbfileid
_s2member_sig
currentpage
archived
toolbox
jahr
mes
page_type
annee
sub_id
responsive
rewritemime
cal
no_expand
wptouch_switch
province
rubrique2
s_id
industry
package
val
privacy
callback
inc
ei
b_start:int
ao
reply
affiliation
fmt
moduleid
hkey
_s2member_vars
10
xg_source
recipients
profiletab
updated_after
gallery_id
daterange
g2_return
lcid
product-id
search_author
a_id
cateid
sw
city_id
backlink
property
fb_action_ids
warning
encode
a_bid
rubrique4
titel
libs
fb_action_types
art_id
nombre
forumids
rubrique3
curid
force
foro
active
pristupnost
delvtype
rand
via
cx
id_news
firstname
listtype
maxprice
sanitize
store_code
medium
objecttype
prod_id
smd_process_download
subject_id
nojs
ukey
ceid
keyfield
account_id
review
nggpage
authorid
keepthis
vp_page
cmpid
fname
ldtype
ec
msclkid
__popuppage
id_cat
contact_id
listorder
clang
cache
sectionmax
fulltext
itmid
offmarket
wpfpaction
ad_id
deviceid
program
sec_id
obid
adults
searchinitial
touid
od
sno
listorderby
ucat
pagelayer-template
categorylayout
difftype
search_in
search_keyword
attribute_pa_color
artspec
intruder_1dcca23355272056f04fe8bf20edfce0
gr
showoldetails
date1
forward
terms
mp
event_date
elq
sessionid
11
buttons
referral
rendermode
page4
up
clientzone
_z
ort
page_name
submit_search
tn
testimonials
np
subpage
price_id
sort_on
adv
g2_subview
sector
entry_id
web
communityjobs
forceview
opc
catids
bp-auth
search_in_description
sig
etype
lingua
session_id
sc_word
sc_area
nc
errorcode
middlename
grp
network
pref
akce
bgcolor
bigclass
en
max_id
utm_expid
zenid
tema
category_code
con
bg
12
sequence
language_id
dlm_download_category
vt
meili_aktuell
tmplaceref
rubrique5
dr_resume_button
portal
dr_resume_link
preacher
cal_m
lt
hood
b_id
lay
c2
product_code
cal_y
job_id
cm_mmc
icf
cmp
virtuemart_product_id
cm
foogallery
icid
res
role
acc
pageaction
kategori
feature_id
market
num1
widgettype
ptid
submenu
curpos
x-client-sku
_cclid
guide
categoryids
for
dept
rubrique6
gubun
id_currency
searchstring
x-client-ver
prefix_id
tr
startrow
icn
can_id
cookies
lock
titulo
sport
nodeid
prev
submitcurrency
ville
p_auth
filter_tag
bc
num2
sc_id
ll
tube
catgid
nv
qurl
email_subject
email_referrer
zc
fb_source
big
fontstyle
creative
goods
sns
cbskin
disp
sz
rows
selectedpageversions
ap
fileid
au
vendor
n_items
report
goodsno
tribe_events
dynamisch
config
os
click
sntz
_t
_siteurl
sort_field
load
follow_id
list_type
grid-orderby
bbs_code
pcmd
follow_app
follow_area
showagent
story
noticia
cf
max_results
scale
ar
rec
pm
forumid
searchtext
syf
foto
id_lingua
wt.mc_id
datum
modul
brand_id
recent
shop_domain
setlang
gcatid
jump
show_products
search_key
bannerid
ret
idart
auto
course_id
anno
sfiltertags
cityid
linkid
tour
snax_login_popup
mlsid
tool
file_srl
searchterm
sessioncopycomplete
server
mod_id
controlleruri
websitekey
rating_filter
idm
sign
land
lx
me_popup
spage
filtered
_rnd
as
symbol
authors
rubrique7
sort_dir
pro
cof
image_order
prev_no
activetab
state_id
undo
plid
duration
lcp_page0
startdate
dp
13
car
mois
pk_kwd
ismoreexpandedview
length
ai1ec_cat_ids
about
password
undoafter
vd
entries_per_page
mobileversion
wc_view_mode
showcategory
property_id
videoid
price_from
occurrence_id
rowstart
camp
review_hash
category-id
favs
price_to
event_id1
viewall
showproducts
newsessionid
sc_serial_code
attach
fq
pbcatid
test
areaid
search_text
attribute_pa_size
subsubcat
unit
wct
courseid
da_image
idtt
artistid
14
page_number_2
clear_cache
re
gt
noscript
block_id
searchjson
show_all
date_from
br
frame
no_cover
vote
isappinstalled
mandat
search_app
bn
anid
thema
banner_action
arg
board_id
rubrique8
usces_page
longurl
ordertype
qa
usr
dc
local
process
productlinknotfound
makename
ti
loginstatus
redirect_to_referer
showtj
openfile
phpmyadmin
showtype
reload
txt
15
cap_id
pagecomponentid
selectedincentiveid
finalidade
forexarab
lb
homepage
nojsmode
wp
store_id
kboard_content_redirect
ltr
totalresult
dil
stranka
frontpage
id_employee
sfilterdate
scroll
orientation
discipline
rubrique9
ctz
offer_id
catname
fw_sig_url
fw_sig_site
fw_sig_permissions
fw_sig_tier
fw_sig
fw_sig_premium
fw_sig_time
fb_sig_network
fw_sig_social
fw_sig_locale
person
fw_sig_partner
company_id
bt
hide_tag_log
errordesc
background
no_definitions
_gl
sex
dataid
agentid
m1
product_quickview
rok
appointmenttype
pb
storytopic
items
lang_id
ph
groupfilter
direct
isresult
video_id
isnav
clickid
privatefilter
cal_d
xml_sitemap
mediatypeid
search_api_fulltext
mixfilter
n_id
16
inapp
ymd
syfnmb
embedded
lvl
frm
tradenumfilter
last_url
kontakt
nm
gal
group_by
smid
resource
district
colour
sc_level
ipp
header_img
question
brandcode
layer
latitude
suche
events
prt
topmenu
esid
kategoria
sc_article_type
timestamp
viewtype
sc_sdate
reaction
side
cur_code
id_categoria
sc_edate
sc_view_level
i_articleid
sc_serial_number
m2
mv
zx
_fid
phone
cpid
nl
ccm_paging_p
psearch
family
maxrows
uenc
17
jet-woo-builder
zipcode
rubrique10
_cldee
model_id
date_to
last
of
filter_relational_operator
screenname
sj
posts_per_page
doi
enc
20
id1
ida
auto_frame
tconfig
nom
fromforum
vm
out_of_stock
country_id
givedonationforminiframe
longitude
journal
answer
u2
call
reply_comm
gamehall
rubrique11
isallowed
signin
recherche
scat
div
recipientid
scene
kr_mode
discount
row
project_id
ssid
tve
g2_highlightid
handler
filter_id
sv
button
metadata
services
export
client-request-id
reference
file_name
mail
cidade
children
revpage
elqcampaignid
jobtype
goal
bathrooms
mediaid
type_1
openid
nome
obj_prefix
18
doc_id
start_from
langchoice
it
cate_sub_idx
detailid
myord
intro
protypeid
vehicle
overlay
endpoint
hss_channel
rand_id
campaign_id
search_word
disp_order
forgot
pricefilter
ev
event_type
homeid
parentid
listingid
checkin
venue
ai1ec_event
usesamllogin
hp
recommend_type
19
branch
subtype
virtuemart_manufacturer_id
is_mile
script
toggle
fullscreen
contentencoding
myshownums
matchtype
campaignid
webid
emci
directory_radius
rubrique13
brandid
emdi
product_type
selected_section
mk
searchkeyword
chronoform
sprache
urlpath
catecd
asset_id
xn_auth
is_gndl
_locale
sq
cnt
brands
artikel
hashed
log
ot
provincia
tit
block
sslid
id_shop
rubrique12
_paymenttype
rn
board_cate
partnerid
features_hash
live
element
widgetid
cntnt01origid
dclid
cat1
idd
searchdetails
familyid
activity
dep
forceversion
recaptcha-opt-in
reppost
avia_forced_reroute
srt
th
module_srl
includeblogs
zid
t1
produkt
search_field
obj
verb
font-size
overview
acpage
schoolid
dopost
help
g2_page
specialfilter
documentid
fkeyid
town
record
sub1
listingsource
match
authorspage
gift
itm_source
itemtype
idnews
21
_pg
widget
rubrique15
cal_id
all_activity
rh
board_idx
tok
man
paginator-page
wantsurl
rubrique14
live_configurator_token
dept_id
id_attachment
listid
team_id
hotel
goods_id
search_category
club
intcmp
affiliate_id
subject:list
memid
filter_0
modelo
area_id
stream
idt
ctx
busca
internetprice
zoek
cbfid
akey
pre
root
yclid
tsmac
filtertype_0
tsmic
portfolioid
dm_i
qbhf
sale_lease
pd
icc
uselang
transmission
design
22
categoryname
tagsrenderer-tag
enddate
page5
exp
wbnewsid
_np
trespass
caa_id
bdid
email_id
pricerangefrom
pricerangeto
ssearch
sg_type_pg
publicationid
topics
accion
pfc
numero
rubrique
infoid
soldproperty
affiliateid
in_stock
page_number_3
pass
rubrique16
is_pcver
order_status
inheritredirect
pgflngid
titre
srss
issn
23
orden
mailpoet_router
searchpage
subdomain
prid
ivrating
read
auteur1
affcode
groupyear
options
regkey
engine
katid
groupbyfield
groupmonth
send
links
rubrique17
set-aa
qt
stage
skey
search_keywords
organization
utm_keyword
listingsortoptions
feed_id
proid
satom
porto_builder
tracking
why
25
page_number
s_url
el_dbe_page
or
loginfacebook
incentiveaccountid
tcbf
poll
edition
zoneid
mailid
meta
scategory
signup
oldal
referrer_id
cr
page_id_all
smode
su
sort_direction
notfound
wppa-occur
myboard_code
application
id_noticia
showsoldmessage
presetparams
ascampaignid
asfrontpageid
bannerclickid
acao
rep
filter_1
search_term
promocodeaction
eventdate
listing
xedm
calendarmonth
property_type
rb_br
ticket
productlistview
rubrique18
loginuserid
article1
misto
_kw
features
social_action
social_controller
cls
girl_id
bk
rootfolder
24
nb
grid
ntype
navigationid
enmse
rank
coverage
ag
gp
pcid
active_action
rooms
rubrique20
special_version
api
region_id
init
galleryid
quot
itm_campaign
type2
m_id
null
posted
par
idsite
end_date
rubrique19
enttype
marka
trim
article3
panel
point
sh
geo
station
_au
awc
kod
refurl
starturl
nights
enmse_mid
sc_order_by
featureor
gr_id
filtering
custombox
mapdata
mstrcode
impressum
al
sort_id
pager
sc_lang
sg
openform
referrerquerystring
letra
simple
utm_name
left
v1
sortcolumn
btn
addnewcustomer
loginredirect
office
years
demo
filtertype_1
loop
article4
gameid
authuser
vol
cat2
bbs_id
vc
theme_font
publisher
s3
ref_id
minacres
26
serial
topnav
sbp
apply
attribute_size
serie
comment_id
banner_id
smallclassname
intarchcatid
no_lredirect
autostart
is
access
pkg
history
proptype
oaparams
searchform
char
integration
online
rcid
showlogin
delay
imageid
px
companyid
rcp_action
wysija-page
rubrique21
topage
copyright
logoutgoto
sharelink
oe
focus
recordcode
id_page
filtername
article2
userkey
starttime
tbl
30
location_name
nv_redirect
long
jumpurl
theme_id
variation
clickthrucustomernumber
clickthruemail
ref_url
playlist
nt
hsa_ver
article8
people
buyproducts_id
bereich
idtypebien
reduced
ged
service_id
hsa_grp
galleries
27
hsa_cam
useglobalvehicleidmatch
hsa_acc
hsa_ad
article6
article7
sshowas
padding
hsa_net
idtipologia
primaryid
ff
pfstyle
swcfpc
plan_id
qa_1
rubrique22
article5
yearbuilt
agent_id
custom
recentchanges
id_menu
startmonth
showid
statfrom
pcomsession
footer
newlang
filterquery
filter_material
url_locale
column
speaker
bukva
hsa_src
walkscore
libraryitemid
maxacres
minfootage
pagen_3
itm_medium
article13
ui
chan
id_property_type
t2
schools
xld_rct
article14
sso
switched
surname
cname
jump_from_id
rubrique23
mnid
ncid
communities
kal_date
quality
badge
category1
dur
dashboardid
maxfootage
ok
hideimages
postcode
toggleopen
ab
referrerguid
filter_2
mediado
logo
exam
staff
hit
mylogout
locations
filtro
include_category
inf_contact_key
directory
abstract
article9
am
28
communitykey
strack
article10
proc
product_brand
status_id
responsemoduleid
show_id
attachedto
wwwredirect
date_end
hsa_tgt
ssa_locale
xtor
spid
tempid
contactres
realm
hsa_mt
hsa_kw
product_name
coppa
submit.x
29
search_target
searchkey
latest
media_category
recruiter_id
box
saison
esrc
isc
q2
cad
buy
31
bd
fb
submit.y
app_id
mnuid
hide_subsequent_recurrences
gc
id_post
span
robots.txt
rubrique24
todo
sort1
startpage
subcatid
142
stext
bodystyle
p3
rubrique26
article18
wrap
phrase
_v
fd
filtertype_2
_ke
monat
input
shortlink
article12
curr
bukken
sname
bl
mb_id
stat
startitem
article17
expanded
preload
rent
globalvehicleid
showcontactform
cy
markers
sms
article11
ex
linkcode
newstypeid
globalvehicletrimid
tiraz
schooldistrict
page_size
redirectparameters
_lid
autoexpanddisclaimer
fm
showincentivedetails
showseetrimslink
verified
errmsg
urlid
pv
shopid
t302
wpam_id
tl
numberitems
ident
promotion
accent_color
selected_size
wppa-album
df
layerid
152
plz
wkst
wc-api
slc_lang
id_auteur
exclusiveno
f1
dum
article16
sortdir
article15
current_page
show_form_title
nmonth
show_form_description
prefn1
32
byp455
prefv1
gh_jid
amnoroute
wmode
cur_cc
rubrique27
gl
dbutton
product_category_depth1
product_category_depth2
pbutton
per
goback
obutton
product_category_depth3
product_category_depth4
search.x
subseason
ai
ob
utm_id
click_id
lr
showsignup
aktion
click_source
suid
ecid
il
sgid
cg
lnk
page_number_4
pubid
intruder_3cd5eb86aae6d8e5821dbbe63f4cc302
v2
add2basket
weight
s_kwcid
im
idcategory
webhost
booking_url
dia
132
uname
rm
work
artnr
orderdir
frame-nonce
booking_title
first_char
booking_post_id
marque
de
comp
35
rubrique30
gi
fref
hs_preview
rubrique29
prov
filter_attr
campid
rubrique25
audience
site_return_url
m22pageno
smagentname
smauthreason
groups
tz
34
produto
cta
_is
siteurl
33
arch
linktype
after
job_type
change
t_id
unknown
summary
realmoid
spec
action_object_map
placement
pricehigh
ctg
ssl
article23
uniqueid
tg
classname
animalid
listingitemid
returl
mime
index_id
subscribe
zona
subscribeurl
g2_gallerysid
id_article_media
lmid
ck
article21
board_name
calendar_id
rubric
em
gotopage
coupon_code
sort_bef_combine
hotel_id
newsletter
rfrom
ipage
entityid
fmdo
162
us
_p
ai1ec_tag_ids
knr
li
trade
datefromyear
depart_id
article20
article22
tab_id
create
datefrommonth
commit
datefromday
datetoday
datetomonth
datetoyear
clientid
param1
markerstyles
filter_3
rubrique31
es
commentpage
n_plain
172
venoscript
article19
bookid
galleyfulltext
mcid
partner_id
rubrique28
article28
sobi2task
classicview
filter_type
modelid
po
search.y
withoutcomments
filter_name
article26
startindex
accessibility
_dnlink
really_curr_tax
list_mode
dom
dr
paging
df_id
article24
regionid
buscar
caldate
biblionumber
37
_act
eventkey
i_dsc
afmc
article30
fw_comments_order
fw_comments_page
taal
action_type_map
secure
title_id
movie
room-type
dasi
article27
36
generations
article25
article29
mh
id_art
i_dsc_and
enmse_spid
sortpath
authstate
contentpageid
advanced_search
enmse_sid
criteria
selection
sort2
nyr
fr_id
viewdate
formtype
rubrique34
jsessionid
servid
sdate
section2
_k
suppfiles
var_lang
40
book_id
nowap
searchjournal
filtertype_3
listingtype
sfilterauthor
displaymode
orgid
showpage
fa
38
cv
newsview
do_pdf
article32
addargs
vdid
blogtype
enmse_o
rpi_click
random_id
pricelow
yy
c3
agency
purpose
rootid
enmse_av
repquote
selectedfilters
who
compositetype
stars
idb
krypto
el_mcal_month
sess
prompt
cate1
card
rubrique33
article31
splash-name
attribute_color
el_mcal_year
dbuje
filter_colour
indexterms
ep
un
_s
layoutmode
match_option
d1
collapse
filtre
action_ref_map
rubrique32
news_aggregator
rubrique36
cisloclanku
rubrique38
rubrique35
category_alpha
212
agenthomepagesearchtype
facet
sezione
__mode
utmsource
chartwidth
connexion
sc_word2
fromid
sbutton
category_header_img
sc_andor
words
qblog
click_from
checkadd
checkip
doip
adcode
lfid
rcstdid
apid
display_faq
_psq
mdid
calendarid
hashtag
sale_date
slot
grupo
article35
i_id
out
singleproduct
article38
pagecode
view_id
grade
evento
showtitle
article33
idgallery
recent_topics_start
x-amz-algorithm
x-amz-credential
originalid
x-amz-date
x-amz-signature
picid
x-amz-signedheaders
islem
rubrique37
system
modlg
bnr
_82_redirect
razdel
x-amz-expires
fecha
me
p_p_cacheability
referid
temp
__utma
finduserthreads
inc_subcat
ispage
filter_type_1
subj
poid
filter_value_1
teamid
filter_field_1
titolo
guest
enmse_c
enmse_p
gmn
exe
product_list_order
validationmessage
gallery_link_id
ddownload
notice_id
ht
__utmc
__utmz
gateway
42
postalcode
y_source
__utmb
article34
gpc
game_id
ld
nlt
frompage
50
39
minor
fi
slider
srch
story_id
var_mode
merit
var1
article48
article40
website
41
int
pan
pattern
idcible
type_2
44
merk
source_url
public
catoid
clients
reply-to
codice
fpage
nonce_token
next_page
article39
heading
related
galerie
charset
how
compid
article36
content_type
listingdistricttypeid
namespace
sendpass
loggedout
formats
cartid
pro_id
article50
article37
notice
sp_month
sp_year
request_locale
run
article51
fbrefresh
tribe_events_cat
article41
subcats
vistas
locale-attribute
wm
article43
article46
_mid_
article47
dstctnid
43
metadatalink
move
picture
bnq
edate
hints
related_post_from
node_id
te
setlanguage
pagenr
tvet
rubrique39
paginate
cbexit
article53
writer

DiamondSorter_Chat/config.py
import enum
SERVER_URL = "http://104.156.244.168:5000"
class ClientEvent(enum.Enum):
    logout = 0 #{"message", "status"}
    get_contacts_info = 1 #{"friend_list":[{"nickname", "user_id", "username","messages":[{"content","username"(me:None),"timestamp"}]}],"group_info":{group_id:{"name","manager_username","members": [{"user_id", "nickname", "username"}],"messages":[{"content","username"(me:None),"timestamp"}]}}}
    friend_request = 2 #{"friend_requests": [{"user_id","username","nickname"}]}
    login_error = 3 #"message","status"
    add_friend_error = 4 #"message","status"
    add_friend_request_success = 5#"message","status"
    get_friend_request = 6#"message","status", "user_id", "username", "nickname","avatar"
    accept_friend_request_error = 7#"message","status","username"
    accept_friend_request_success = 8#"message","status","username"
    get_friend_accept = 9#"message","status", "user_id", "username", "nickname","avatar"
    create_group_success = 10#"message","status","group_id","group_name","members":[{"username","nickname"}]
    create_group_error = 11 #"message","status"
    join_group_request_success = 12#"message","status"
    join_group_error = 13#"message","status"
    group_request_all  = 14 #{"group_requests":[{"user_id","username","nickname","group_id","group_name"}]}
    get_group_request_accepted = 15#"message","status", "group_id","group_name","timestamp","manager_username"
    accept_group_request_error = 16#"message","status","group_id","username"
    accept_group_request_success = 17#"message","status","group_id","username"
    send_message_success = 18#"message","status","md5"
    send_message_error = 19#"message","status","md5"
    receive_message = 20#"username","content","nickname","timestamp",("group_id")
    get_group_request = 21 #"message","status", "user_id","group_id","username","nickname","avatar"
    get_group_info = 22 #"message","status", "group_id","group_name","manager_username","members":[{"user_id","username","nickname"}],"messages":[{"content","username"(me:None),"timestamp"}]
    new_user_join_group = 23 #"message","status", "group_id","group_name","user_id","username","nickname"


class ServerEvent(enum.Enum):
    "/login"#json "message","status",状态码 ("session_id","user_id","username","nickname") 
    "/register"#json "message","status",状态码
    "connect"#websocket  username, session_id, timestamp
    "disconnect"#websocket
    request_add_friend = 0 #user_username, friend_username
    accept_friend_request = 1 #username, friend_username, isaccepted
    create_group = 2 #username, group_name
    request_join_group = 3 #username, group_id
    accept_group_request = 4 #username, group_id, isaccepted
    send_message = 5 #username, content, group_id, friend_username, md5
    delete_friend = 6 #username, friend_username
    delete_group = 7 #username, group_id
    leave_group = 8  #username, group_id
    getGroupInfo = 9 #username, group_id

class Status(enum.Enum):
    SUCCESS = 0
    REJECTED = 1
    ERROR = 2
    OUTDATED = 3
    DUPLICATE = 4
from PyQt5.QtCore import QStandardPaths
CACHE_DIR = QStandardPaths.writableLocation(QStandardPaths.CacheLocation)
DEFAULT_CONTACT_AVATAR = ":/images/DefaultContactAvatar.png"
DEFAULT_GROUP_AVATAR = ":/images/DefaultGroupAvatar.png"